### PR TITLE
License

### PR DESCRIPTION
**Pull request by ahgittin**
minor edits to license metadata

### CODE REVIEW

**Comment by ahgittin** [comment]
failure is unrelated (karaf issue people are working on).

@johnmccabe does this look okay to you?

File: usage/jsgui/src/main/webapp/assets/js/libs/URI.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Taken from http://code.google.com/p/js-uri/ r29.  BSD license.
+ * Taken from http://code.google.com/p/js-uri/ v0.1.  BSD license.
```
Comment (johnmccabe) [inline usage/jsgui/src/main/webapp/assets/js/libs/URI.js]: Looks good, triple checked the version in the 0.1 release is identical

File: usage/cli/src/main/license/source-inclusions.yaml [hunk 2]
```diff
@@ -20,5 +20,5 @@
 # extras file for org.heneveld.license-audit-maven-plugin
 # listing projects from which *source* files are included
 
-- id: underscore.js
+- id: underscore.js:1.7.0
```
Comment (johnmccabe) [inline usage/cli/src/main/license/source-inclusions.yaml]: Just a query on the plugin - I can see this ties up with the entry in the overrides.yaml, its not clear how/where this file gets picked up from the [license-audit-maven-plugin](https://github.com/ahgittin/license-audit-maven-plugin) readme?

Comment (ahgittin) [inline usage/cli/src/main/license/source-inclusions.yaml]: our `make-*-license*` scripts combine the relevant `source-inclusions.yaml` files in order to construct the `extrasFile` passed to the plugin.  i've updated `README` to explain this.

**Comment by ahgittin** [comment]
@johnmccabe great comments -- many eyes is great for making this rock solid!  i think i've addressed them, and regenerated licenses.  ready to merge?

File: LICENSE [hunk 3]
```diff
@@ -352,6 +352,14 @@ This project includes the software: underscore.js
   Used under the following license: The MIT License (http://opensource.org/licenses/MIT)
   Copyright (c) Jeremy Ashkenas, DocumentCloud Inc (2009-2013)
 
+This project includes the software: underscore.js:1.7.0
+  Available at: http://underscorejs.org
+  Developed by: DocumentCloud Inc (http://www.documentcloud.org/)
+  Inclusive of: underscore*.{js,map}
+  Version used: 1.7.0
+  Used under the following license: The MIT License (http://opensource.org/licenses/MIT)
+  Copyright (c) Jeremy Ashkenas, DocumentCloud Inc (2009-2014)
```
Comment (johnmccabe) [inline LICENSE]: (c) 2009-2014 Jeremy Ashkenas, DocumentCloud and Investigative Reporters & Editors

See [underscore-min.js](https://github.com/apache/incubator-brooklyn/blob/master/usage/cli/src/main/resources/brooklyn/item-lister/statics/style/js/underscore-min.js)

File: usage/dist/licensing/overrides.yaml [hunk 4]
```diff
@@ -176,6 +200,15 @@
   notice: Copyright (c) Jeremy Ashkenas, DocumentCloud Inc (2009-2013)
   license: MIT
 
+# used in CLI (and in docs)
+- id: underscore.js:1.7.0
+  version: 1.7.0
+  files: underscore*.{js,map}
+  url: http://underscorejs.org
+  organization: { name: "DocumentCloud Inc", url: "http://www.documentcloud.org/" }
+  notice: Copyright (c) Jeremy Ashkenas, DocumentCloud Inc (2009-2014)
```
Comment (johnmccabe) [inline usage/dist/licensing/overrides.yaml]: (c) 2009-2014 Jeremy Ashkenas, DocumentCloud and Investigative Reporters & Editors

See [underscore-min.js](https://github.com/apache/incubator-brooklyn/blob/master/usage/cli/src/main/resources/brooklyn/item-lister/statics/style/js/underscore-min.js)

**Comment by johnmccabe** [comment]
Just that last comment on the underscore copyright and it looks good to me.

**Pull request merged**